### PR TITLE
#160252034 Enable users perform crud operations on the fixtures

### PR DESCRIPTION
**Pull request by kaguna**
#### What does this PR do?
- Enable users perform crud operations on the fixtures.

#### Description of Task to be completed?
- Write all crud functions on the controller.
- Connect the via routes.
- Create bot to fake the data.

#### PT story
[#160252034](https://www.pivotaltracker.com/story/show/160252034)

### CODE REVIEW

File: spec/requests/test.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        players_id: "1",
+        fixtures_id: "1"
+    }
+  }
+
+  describe "POST event/create" do
+    context "when the request is valid" do
+      before { post "/#{team_id}/#{player_id}/#{fixture_id}", params: event_params }
+      it "creates a new event" do
+        puts event_params, "@@@@@@@@@@@@@@@@@@@"
```
Comment (kevinsamoei) [inline spec/requests/test.rb]: Remove the puts

File: spec/requests/test.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        expect(json["first_name"]).to eq("James")
+      end
+      it "returns status code 201" do
+        expect(response).to have_http_status(201)
+      end
+    end
+
+    # context "when the request is invalid" do
```
Comment (kevinsamoei) [inline spec/requests/test.rb]: Also remove the commented out code

File: app/controllers/fixtures_controller.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
+class FixturesController < ApplicationController
```
Comment (kevinsamoei) [inline app/controllers/fixtures_controller.rb]: This file is not tested, it has 0% coverage

**Comment by kaguna** [comment]
@kevinsamoei I have tested the controller and made the changes as requested.

**Pull request merged**